bundles/org.openhab.binding.smsmodem/src/main/java/org/openhab/binding/smsmodem/internal/handler/SMSConversationHandler.java

```diff
@@ -0,0 +1,152 @@
+        if (conversationRecipient.equals(sender)) {
+            updateState(SMSModemBindingConstants.CHANNEL_RECEIVED, new StringType(text));
+        }
+    }
+
+    protected void checkAndUpdateDeliveryStatus(String messageRecipient, DeliveryStatus sentStatus) {
+        String conversationRecipient = config.recipient.trim();
+        // is the recipient the one handled by this conversation ? :
+        if (conversationRecipient.equals(messageRecipient)) {
+            updateState(SMSModemBindingConstants.CHANNEL_DELIVERYSTATUS, new StringType(sentStatus.name()));
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            return;
+        }
+        if (channelUID.getId().equals(SMSModemBindingConstants.CHANNEL_SEND)) {
+            send(command.toString());
+            updateState(SMSModemBindingConstants.CHANNEL_SEND, new StringType(command.toString()));
+        }
+    }
+
+    public void send(String text) {
+        SMSModemBridgeHandler bridgeHandlerFinal = bridgeHandler;
+        if (bridgeHandlerFinal != null) {
+            bridgeHandlerFinal.send(getRecipient(), text, config.deliveryReport, config.encoding);
+        } else {
+            logger.warn("Only channel 'send' in SMSConversation can receive command");
+        }
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(SMSConversationConfiguration.class);
+        bridgeHandler = getBridgeHandler();
+        setStatusByBridgeStatus();
+    }
+
+    private void setStatusByBridgeStatus() {
+        SMSModemBridgeHandler bridgeHandlerFinal = bridgeHandler;
+        if (bridgeHandlerFinal != null) {
+            switch (bridgeHandlerFinal.getThing().getStatus()) {
+                case INITIALIZING:
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+                    break;
+                case OFFLINE:
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+                    break;
+                case ONLINE:
+                    updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE);
+                    break;
+                case REMOVED:
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+                    break;
+                case REMOVING:
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+                    break;
+                case UNINITIALIZED:
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+                    break;
+                case UNKNOWN:
+                    updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.BRIDGE_OFFLINE);
+                    break;
+            }
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+        }
```

Comment:
The Thing status is set by the framework automatically according the bridge status, if you set the Thing status to UNKNOWN in `initialize()`. Then, this could be removed.

Reply:
I made a test and if I set the child handler status to "UNKNOWN" and leave if for the Bridge to update (deleting setStatusByBridgeStatus() method), it stays "UNKNOWN", even if the bridge status switch to "online". 
Or maybe I didn't understand your proposition ?

By reading code, the bridgeStatusChanged() method of BaseThingHandler effectively change the child handler status to "ONLINE" when the bridge get online, but only if the last status details was BRIDGE_OFFLINE

Reply:
You still need to set the child Thing ONLINE, if you think it is online. In your case this might be always the case, if the bridge is online. So, you can set it ONLINE in `initialize()` directly. Then, it will be online if the bridge is online and offline if the bridge is offline.

You only need to set it to UNKNOWN (and later to ONLINE) if you don't know the state in `initialize()`, yet.

Reply:
Ah, I get it now !
Indeed, I can set it to ONLINE directly.
Done, thanks

By the way, there is one drawback : the bridge start as "unknown" because it has to establish connection. But the smsconversation thing is immediately online at this time (and the status doesn't reflect the reality : it is not fully functionnal because of the bridge).

Reply:
Just tested your code and a reason why it doesn't work as expected is that you don't set the ThingStatus in the bridge's `initialize()` to UNKNOWN. It works for me if I do that.

Reply:
![image](https://user-images.githubusercontent.com/1935721/203526641-ab20979e-5b37-4803-99e0-9ecf665fe7e4.png)

Strange, it doesn't work for me. When the bridge is initializing (unknown), the child conversation thing is online. Maybe an order initialization issue ?
But it is no big deal, I'm OK with that, so I let you decide if it needs further investigation.

Reply:
I just tested it again and you're right, as long as the bridge is in UNKNOWN state, the child Things are ONLINE. Only if the bridge is set to OFFLINE, the child Things are set to OFFLINE, too. Normally you wouldn't notice, because the bridge stays in UNKNOWN state only for very short time. In this case the timeout is quite long.

I would expect the SMS modem to respond within subseconds and would chose a shorter timeout, anyway. But maybe there are things I didn't consider.

I merge it for now. If you want to improve the behavior, you could file a follow-up PR.